test/docker/sql/sql.go

```diff
@@ -0,0 +1,85 @@
+package sql
+
+import (
+	"database/sql"
+	"fmt"
+	"time"
+
+	"github.com/ory/dockertest/docker"
+
+	patronDocker "github.com/beatlabs/patron/test/docker"
+	// Integration test.
+	_ "github.com/go-sql-driver/mysql"
+	"github.com/ory/dockertest"
+)
+
+const (
+	dbHost           = "localhost"
+	dbSchema         = "patrondb"
+	dbPassword       = "test123"
+	dbRootPassword   = "test123"
+	dbUsername       = "patron"
+	connectionFormat = "%s:%s@(%s:%s)/%s?parseTime=true"
+)
+
+// SQL defines a docker SQL runtime.
+type sqlRuntime struct {
+	patronDocker.Runtime
+}
+
```

Comment:
If we are not making the ports dynamic, maybe good to add a comment that we cannot run tests in parallel.

Reply:
since i moved everything to the test package we run kafka and mysql only once. so testing in parallel is doable.